guides/RELEASE-PROCESS.md

```diff
@@ -2,32 +2,28 @@
 
 1. **Create a release commit**
 
-    1. Get the latest from the `master` branch
-         ```
-        git pull
-         ```
-    1. Check the status of your working directory to ensure that you don't have any local changes. If you do have local changes [stash or discard](https://docs.gitlab.com/ee/topics/git/numerous_undo_possibilities_in_git/#quickly-save-local-changes) them before going to the next step.
-         ```
-        git status
-         ```
+   1. Checkout the latest `master` branch and ensure that you don't have any local changes. If you do have local changes [stash or discard](https://docs.gitlab.com/ee/topics/git/numerous_undo_possibilities_in_git/#quickly-save-local-changes) them before going to the next step.
+      ```
+      git pull && git status
+      ```
    1. Run the test suite and ensure JS unit tests, JS linting, and Sass linting pass locally.
-         ```
-         yarn test
-         ```
-   1. Run the prepublish script from the root of your local git repo. This will bump the version, build the documentation, and transpile + bundle `dist` assets.
       ```
-      ./scripts/prepublish.sh
+      yarn install && yarn test
       ```
-
-   1. Add, commit and push to `master` with a new version number, replacing v1.1.0 with your version:
+   1. Check for unexpected visual regressions. The app must be running locally in order for the tests to run, so run `yarn start` if the app isn't up already.
       ```
-      git add --all
+      backstop test
       ```
+   1. Run the prepublish script from the root of your local git repo. This will bump the version, build the documentation, and transpile + bundle `dist` assets.
       ```
-      git commit -m "Release v1.1.0"
+      ./scripts/prepublish.sh
       ```
+   1. Create a new branch for the new release, commit the changes and push the branch up. Follow the normal PR process for reviewing and merging the branch into master. In this example replace v1.1.0 with your version:
```

Comment:
@line47 @pwolfert I'm thinking it would be nice for this to be handled by `prepublish.sh` too, but I'm not sure if that's too prescriptive. Thoughts?

Reply:
It's certainly a lot of lines to be copying and pasting into your terminal and has multiple opportunities for mistakes. I agree that it should go in a script. Putting it in `prepublish.sh` makes sense. I don't know if that script name does it justice, though, for what it's doing. At that point it's creating the release branch as well, so it's more about building a release that will be on GitHub. What if it were renamed to `create-release.sh` or something? You _create_ the release, then you _publish_ the release (and you also merge the release branch).

Reply:
Either way, I agree